client/lib/wrap-es6-functions/index.js

```diff
@@ -0,0 +1,25 @@
+import partial from 'lodash/partial';
+
+function wrapFnWithWarning( fn, name ) {
+	const consoleFn = ( console.error || console.log ).bind( console );
+	return function() {
+		const err = new Error( `${ name } is not supported on all browsers. You must use a replacement method from lodash.` );
+		consoleFn( err );
+		return fn.apply( this, arguments );
+	}
+}
+
+function wrapObjectFn( obj, objectName, key ) {
+	Object.defineProperty( obj, key, { value: wrapFnWithWarning( obj[ key ], `${ objectName }${ key}` ) } );
+}
+
+export default function() {
+	[ 'keys', 'entries', 'values', 'findIndex', 'fill', 'find' ]
+		.map( partial( wrapObjectFn, Array.prototype, 'Array#' ) );
+
+	[ 'codePointAt', 'normalize', 'repeat', 'startsWith', 'endsWith', 'includes' ]
+		.map( partial( wrapObjectFn, String.prototype, 'String#' ) );
```

Comment:
I think it was the tests that were borked, I was stubbing the methods instead spying. The tests are OK now.